src/Symfony/Component/Serializer/Attribute/SerializedPath.php

```diff
@@ -18,27 +18,43 @@
 /**
  * @author Tobias Bönner <tobi@boenner.family>
  */
-#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY)]
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY | \Attribute::IS_REPEATABLE)]
 class SerializedPath
 {
     private PropertyPath $serializedPath;
 
+    private array $groups;
+
     /**
-     * @param string $serializedPath A path using a valid PropertyAccess syntax where the value is stored in a normalized representation
+     * @param string          $serializedPath A path using a valid PropertyAccess syntax where the value is stored in a normalized representation
+     * @param string|string[] $groups         The groups to use when serializing or deserializing
      */
-    public function __construct(string $serializedPath)
+    public function __construct(string $serializedPath, string|array $groups = ['*'])
     {
         try {
             $this->serializedPath = new PropertyPath($serializedPath);
         } catch (InvalidPropertyPathException $pathException) {
             throw new InvalidArgumentException(\sprintf('Parameter given to "%s" must be a valid property path.', self::class));
         }
+
+        $this->groups = ((array) $groups) ?: ['*'];
```

Comment:
I've tried that but it led to more failing tests. So I reverted it back until we decide what it could actually mean, so we could change the tests.
In my point of view, empty array here is nonsense, as it means "apply serialized name/path to **no groups**".
As alternative to fallback to `['*']`, we can throw exception. WDYT?